front/src/components/catalog/CatalogStyles.ts

```diff
@@ -20,13 +20,38 @@ export const useStyles = createStyles((theme) => ({
         [`@media (min-width: ${theme.breakpoints.xl}px)`]: {
             gridTemplateColumns: "auto auto auto",
         },
+
+        [`& .${getRef("card")}`]: {
+            height: theme.breakpoints.xs / 2,
+        }
     },
+    catalogOfFour: {
+        display: "grid",
+        gridTemplateColumns: "auto",
+        gap: theme.spacing.md,
+        justifyContent: "center",
+        alignItems: "flex-start",
 
-    card: {
-        marginTop: "20px",
         [`@media (min-width: ${theme.breakpoints.sm}px)`]: {
-            marginLeft: "10px",
-            marginRight: "10px",
+            gridTemplateColumns: "auto auto",
+        },
+        [`@media (min-width: ${theme.breakpoints.md}px)`]: {
+            gridTemplateColumns: "auto auto auto",
+        },
+        [`@media (min-width: ${theme.breakpoints.lg}px)`]: {
+            gridTemplateColumns: "auto auto auto auto",
+        },
+        [`@media (min-width: ${theme.breakpoints.xl}px)`]: {
+            gridTemplateColumns: "auto auto auto auto",
+        },
```

Comment:
I don't think you need this, since xl breakpoint will take styles of lg automatically, unless specified differently.